browser/Makefile.am

```diff
@@ -429,6 +429,7 @@ COOL_JS_LST =\
 	src/control/jsdialog/Util.OnDemandRenderer.ts \
 	src/control/jsdialog/Util.ScrollableBar.ts \
 	src/control/jsdialog/Util.ScrollIntoView.ts \
+	src/control/jsdialog/Util.Shortcuts.ts \
```

Comment:
Unfortunately we don't use any modules and bundler but IIFE approach to merge bundle. You need to put Util.Shortcuts.ts file above Definitions.Menu.ts so they will be already initialized.

Reply:
it's duplicated now :)